chart/k8gb/values.yaml

```diff
@@ -12,6 +12,11 @@ k8gb:
   # -- whether it should also deploy the gslb and dnsendpoints CRDs
   deployCrds: true
   # -- whether it should also deploy the service account, cluster role and cluster role binding
+  feature:
```

Comment:
Why do we need an additional `feature` hierarchy for this one?
why not just `dynamicZones: false` instead of 
```
feature:
  dynamicZones: false
```
?

Reply:
I think it's a noble goal to reflect that, but in practice when this functionality becomes closer to default, it will lead to migration pain to the users.

I suggest to do a single level and reflect maturity in comment. Something like
`dynamicZones: false # Alpha feature`